com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Components/Tonemapping.cs

```diff
@@ -20,7 +20,7 @@ public enum TonemappingMode
         Neutral,
 
         /// <summary>
-        /// Close approximation of the reference ACES tonemapper for a more filmic look.
+        /// ACES tonemapper for a more filmic look.
```

Comment:
to clarify, it mean we don't have approx anymore for ACES? (guess you told me but I forget), does it mean we have a perf hit here?

Reply:
We have both now, approx or not depends on a checkbox (scroll a bit down the file for the checkbox)

Reply:
By default the checkbox is off, no need for anything to do when upgrading. Isn't it more a what's new?